Jenkinsfile

```diff
@@ -183,3 +186,12 @@ def publishPackage(String artifactoryRoot, String deb, String rpm)
 		}
 	}
 }
+
+def getTestDisks()
+{
+	sh "lsblk -f"
+	return sh (
+		script: "sudo fdisk -l | grep 'Disk ' | grep ' 2147483648 bytes' | awk '{print \$2}' | sed 's/://'",
```

Comment:
1. Do we always have 2147483648 bytes here?
2. I think we might need a comment about the purpose of this function for the future

Reply:
Yes, all test images has 2GB disks for test